salt/_pillar/metalk8s_certs.py

```diff
@@ -0,0 +1,51 @@
+import os.path
+import logging
+
+
+SA_PRIVATE_KEY_PATH = "/etc/kubernetes/pki/sa.key"
+
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = "metalk8s_certs"
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def _read_sa_private_key():
+    # FIXME: we only have access to this file because:
+    #   1- Salt master Pod mounts all of /etc/kubernetes
+    #   2- Salt master Pod runs on the 'bootstrap' minion, which is, by chance,
+    #      also the 'ca' minion, the owner of this key
+    # Suggestion: read through https://github.com/saltstack/salt/issues/45882
+    if not os.path.isfile(SA_PRIVATE_KEY_PATH):
+        log.warn("Missing SA key to share with master node.")
```

Comment:
Following #1018 you should maybe start not log directly but put those in `_errors` key but maybe not part of this PR as you want

Reply:
I'll also change the other log (skipping unknown minions) to `debug`